### PR TITLE
back link support

### PR DESCRIPTION
**Pull request by yrong**

### CODE REVIEW

**Comment by yrong** [comment]
backgroud: https://github.com/Requarks/wiki/issues/1998

File: server/models/pages.js [hunk 1]
```diff
@@ -755,10 +768,18 @@ module.exports = class Page extends Model {
         //   }
         // })
         .first()
+      if (page.backLinks && page.backLinks.length) {
+        page.backLinkPages = []
+        for (backLink of page.backLinks) {
+          backLinkPage = await backLink.$relatedQuery('page').column(['id', 'path', 'title'])
```
Comment (regevbr) [inline server/models/pages.js:774]: This can be optimized with Promise.all, no need to wait page by page

**Comment by regevbr** [comment]
@yrong if I understand correctly, you want to provide links to other locals of the current page?
If so, I think it will make more sense to add the locale to the view... Also, you should probably filter the current page from the back links

**Comment by yrong** [comment]
@regevbr  some fix per your suggestion

https://github.com/Requarks/wiki/pull/2023/commits/a5420095f14309544eba2d5f22d0cea2cc7a6d01

**Comment by NGPixel** [comment]
Change target branch to `dev` for this PR.

**Pull request closed without merging**